### ISSUE TITLE
feel uncertain about model robust

### ISSUE THREAD

**Issue by canteen-man** [yuliangguo/Pytorch_Generalized_3D_Lane_Detection#13]
Hi,when i change the calibration of camera,what ever the height or the pitch angel,the model output is totaly error.
Is this algorithm can cover different calibration of camera？
we train this algorithm by usning the real 3d world label with image from different calibration of camera，whether this algorithm can conver different calibration of camera?Or it's cant't to converge directly.

**Comment by yuliangguo** [comment]
Thank you for the question. When you need to re-trained the model on your
own calibration, there are a few that need to be handled properly. First,
all of you data should have the same camera intrinsic parameters, the
algorithm can hardly work with mixed intrinsics parameters. Second, when
you change the height and orientation, you need to input them to the model
for the projection. In other words, you may need other modules to predict
the height and orientation of the camera. And last, you would need to
retrain both the submodels on your own data, combining one pre-trained
submodel with another re-trained submodel may not work.
